lib/background.js

```diff
@@ -26,3 +26,13 @@ chrome.runtime.onMessage.addListener(({ urls }, sender) => {
     });
   });
 });
+
```

Comment:
The discussion in #113 reminded me of this, and I decided to see how Octotree does it. It looks like they went with the second option, and found a way to avoid inserting the script multiple times: https://github.com/buunguyen/octotree/commit/bdfe39510d005240c4011ca6aa8b5761bce71134#diff-d5d296b6086d9d8de73abd3e81e594cdR4

(I'm not planning to work on this, but it might be useful for anyone who is.)